chardet/enums.py

```diff
@@ -31,7 +34,7 @@ class LanguageFilter:
     CJK = CHINESE | JAPANESE | KOREAN
 
 
-class ProbingState:
+class ProbingState(Flag):
```

Comment:
This (and the other ones in this file except for `LanguageFilter`) should inherit from `Enum`, because they're not meaningfully combinable by bitwise operations.

Reply:
That makes sense, I changed this to an Enum in the latest revision.